formats/special_route/publisher/schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "format",
+    "publishing_app",
+    "update_type",
+    "routes"
+  ],
+  "properties": {
+    "base_path": {
+      "$ref": "#/definitions/absolute_path"
+    },
+    "format": {
+      "enum": [ "special_case" ]
+    },
+    "publishing_app": {
+      "type": "string"
+    },
+    "update_type": {
+      "enum": [ "major", "minor", "republish" ]
+    },
+    "title": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "routes": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/route"
+      }
+    }
+  }
+}
```

Comment:
hmm, that was an oversight as well. I guess the fact is that these schemas are not getting used anywhere (neither within this repo, nor by publishing clients), so the bug was not picked up.

Maybe there's a case for adding support for publisher examples as well, I'd be happy to talk through with you what might be involved @elliotcm, or pair on it if you'd like.